cloudinit/config/cc_ubuntu_advantage.py

```diff
@@ -28,6 +31,10 @@
         list is present, any named service will supplement the contract-default
         enabled services.
 
+        In Ubuntu Pro instances, `token` will be ignored. `enable` and
+        `enable_beta` will fully determine what services will be enabled,
```

Comment:
Better readthedocs markdown rendering for static cloud-config keys

We might want to say something about ubuntu-advantage-tool's auto-attach being disabled
if config is provided to cloud-init.
 ```suggestion
        On Ubuntu Pro instances, when ``ubuntu_advantage`` config
        is provided to cloud-init, Ubuntu Advantage Pro's auto-attach feature will
        be disabled and cloud-init will perform the PRO auto-attach, ignoring ``token``.
        The ``enable`` and  ``enable_beta`` values will strictly determine what
        services will be enabled,
```

Reply:
Done.